engines/registry/app/controllers/registry/registry_controller.rb

```diff
@@ -54,9 +54,11 @@ def basic_auth
       # skip authentication if this is not a login request
       return unless request.authorization
 
+      @remote_ip = request.remote_ip
+
       authenticate_or_request_with_http_basic('SUSE Registry Authentication') do |login, password|
         begin
-          @client = Registry::AuthenticatedClient.new(login, password)
+          @client = Registry::AuthenticatedClient.new(login, password, @remote_ip)
```

Comment:
If this instance variable is not being used anywhere, you can just pass `request.remote_ip`.

It is one less variable assignment.

Reply:
The original idea was to use it in other places, good catch